packages/slate-editor/src/extensions/paste-slate-content/PasteSlateContentExtension.ts

```diff
@@ -2,7 +2,7 @@ import type { Extension } from '@prezly/slate-commons';
 
 import { withSlatePasting } from './lib';
 
-export const EXTENSION_ID = 'PasteSlateContentExtension';
+export const EXTENSION_ID = PasteSlateContentExtension.name;
```

Comment:
q: i am not sure, but maybe uglifier can change the name of function and the name can be random

Reply:
Good point. Will fix. Thanks :+1: